pkg/informer/informer.go

```diff
@@ -787,6 +890,96 @@ func newRESTMapper(fn func() (meta.RESTMapper, error)) restMapper {
 	}
 }
 
+type genericInformerBase[InformerType sharedIndexInformerBase, ListerType genericListerBase] interface {
```

Comment:
please put these types at the top, where they are used

Reply:
Done